ev3dev/core.py

```diff
@@ -3046,100 +3144,49 @@ def buttons_pressed(self):
         return pressed
 
 
-class RemoteControl(ButtonBase):
+class RemoteControl(ButtonBase, InfraredSensor):
```

Comment:
I think we should either drop `RemoteControl` and `BeakonSeeker` or keep everything as is. The classes do not do anything that `InfraredSensor` already does not, so why keep them?.

Reply:
agreed they aren't doing much but we need them for backwards compatibility